src/prompt_tasks.py

```diff
@@ -77,11 +82,12 @@ def docker_file_task():
         "It is important to make sure that all libs are installed that are required by the python packages. "
         "Usually libraries are installed with apt-get. "
         "Be aware that the machine the docker container is running on does not have a GPU - only CPU. "
-        "Add the config.yml file to the Dockerfile. "
+        "Add the config.yml file to the Dockerfile. Note that the Dockerfile only has access to the files: "
+        "executor.py, requirements.txt, config.yml, test_executor.py. "
         "The base image of the Dockerfile is FROM jinaai/jina:3.14.1-py39-standard. "
         'The entrypoint is ENTRYPOINT ["jina", "executor", "--uses", "config.yml"]. '
         'Make sure the all files are in the /workdir. '
-        "The Dockerfile runs the test during the build process. " + not_allowed(),
```

Comment:
instead of removing `not_allowed` separate them into must not for `executor` and for `Dockerfile`

Reply:
done